### PR TITLE
fix(Hero): use background instead of backgroundColor to support gradients

### PR DESCRIPTION
**Pull request by marcoskolodny**
Issue: [Link](https://jira.tid.es/browse/WEB-1822)

### CODE REVIEW

File: src/__screenshot_tests__/__image_snapshots__/hero-screenshot-test-tsx-hero-background-brand-in-o-2-new-skin-1-snap.png
Comment (atabel) [inline src/__screenshot_tests__/__image_snapshots__/hero-screenshot-test-tsx-hero-background-brand-in-o-2-new-skin-1-snap.png:1]: no gradient?

Comment (marcoskolodny) [reply src/__screenshot_tests__/__image_snapshots__/hero-screenshot-test-tsx-hero-background-brand-in-o-2-new-skin-1-snap.png:1]: Actually, there is a gradient. It's just not very noticeable. If I pick the topmost color in a color picker tool online, I get this:

image

While the lowest one is this:

image

**Review by pladaria — APPROVED** [review]

File: src/__screenshot_tests__/__image_snapshots__/hero-screenshot-test-tsx-hero-background-brand-in-o-2-new-skin-1-snap.png
Comment (yceballost) [reply src/__screenshot_tests__/__image_snapshots__/hero-screenshot-test-tsx-hero-background-brand-in-o-2-new-skin-1-snap.png:1]: The problem here is that the gradient is starting in the top of the component, the background only have to cover the part of the content to show the gradient correctly

image

@aweell are u agree? or can you see any problem?

**Review by yceballost — CHANGES_REQUESTED** [review]

File: src/__screenshot_tests__/__image_snapshots__/hero-screenshot-test-tsx-hero-background-brand-in-o-2-new-skin-1-snap.png
Comment (marcoskolodny) [reply src/__screenshot_tests__/__image_snapshots__/hero-screenshot-test-tsx-hero-background-brand-in-o-2-new-skin-1-snap.png:1]: I've fixed this issue. Now it should look better :)

**Review by yceballost — APPROVED** [review]

**Review by atabel — APPROVED** [review]

**Pull request merged**